generateBuildrootSBOM.py

```diff
@@ -72,48 +75,54 @@ def create_buildroot_sbom(args, br_bom: cyclonedx.model.bom):
     return br_bom_local
 
 
-def main():
+def my_main():
     parser = argparse.ArgumentParser(description='CycloneDX BOM Generator')
     parser.add_argument('-i', action='store', dest='input_file', default='manifest.csv')
     parser.add_argument('-o', action='store', dest='output_file', default='buildroot_IOT_sbom')
     parser.add_argument('-n', action='store', dest='input_name', default='unknown')
     parser.add_argument('-v', action='store', dest='component_version', default='unknown')
 
     args = parser.parse_args()
-    print('Input file: ' + args.input_file)
-    print('Output BOM: ' + args.output_file)
+    print('Buildroot manifest input file: ' + args.input_file)
+    print('Output SBOM: ' + args.output_file)
     print('SBOM Component Name: ' + args.input_name)
     print('SBOM Component Version: ' + args.component_version)
 
-    # TODO provide a way to specify the meta data of this BOM using cyclonedx-python-lib v4.0.0 schema
-    # which should be managed by cyclonedx.model.bom.BomMetaData() but apparently not yet ready
+    # TODO provide a way to specify the metadata of this BOM using cyclonedx-python-lib v4.0.0 schema
+    # need support to provide the user with the ability to specify the following for this BOM
     # authors
     # component
     # manufacture
     # license
 
-    # TODO determine if we need both a br_bom and a new_bom
-
-    br_bom = cyclonedx.model.bom.Bom()
-    new_bom = create_buildroot_sbom(args, br_bom)
+    br_bom = Bom()
+    new_bom = create_buildroot_sbom(args.input_file, br_bom)
+    br_meta = BomMetaData(manufacture=OrganizationalEntity(name="Acme Inc"))
+    new_bom.metadata = br_meta
+    new_bom.version = args.component_version
 
     # Produce the output in pretty JSON format.
-    from cyclonedx.output import get_instance, BaseOutput, OutputFormat
     outputter: BaseOutput(bom=new_bom) = get_instance(bom=new_bom, output_format=OutputFormat.JSON)
     bom_json = outputter.output_as_string()
     outputfile = open((args.output_file + ".json"), mode='w')
     json.dump(json.loads(bom_json), outputfile, indent=3)
     outputfile.close()
 
-    # Produce the output in XML format.
+    # Produce the output in XML format that is in a one-line format.
     outputterXML: BaseOutput(bom=new_bom) = get_instance(bom=new_bom, output_format=OutputFormat.XML)
     outputterXML.output_to_file(filename=(args.output_file + ".onexml"), allow_overwrite=True)
 
-    from xml.dom import minidom
+    # Produce the output in XML format that is indented format.
     myxmldoc = minidom.parseString(open((args.output_file + ".onexml")).read())
     outputfile = open(args.output_file + ".xml", mode='w')
     print(myxmldoc.toprettyxml(), file=outputfile)
     outputfile.close()
 
 
-main()
+# main()
+
+
+if __name__ == "__main__":
+    import sys
```

Comment:
why the import down here? 
if it was used, than it should have been imported at the start of the file.